string robot_name
float32 x_coordinate
float32 y_coordinate



